src/app/components/elements/panels/UserPassword.tsx

```diff
@@ -99,25 +103,33 @@ export const UserPassword = (
                         </Col>
                     </Row>
                     <Row>
-                        <Col md={{size: 6, offset: 3}}>
-                            <FormGroup>
-                                <Label htmlFor="password-confirm">Re-enter new password</Label>
-                                <Input
-                                    invalid={!!currentPassword && !isNewPasswordConfirmed}
-                                    id="password-confirm"
-                                    type="password" 
-                                    name="password-confirmation"
-                                    onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
-                                        setNewPasswordConfirm(e.target.value);
-                                        setMyUser(Object.assign({}, myUser, {password: e.target.value}));
-                                    }} 
-                                    aria-describedby="passwordConfirmationValidationMessage"
-                                    disabled={!editingOtherUser && currentPassword == ""}
-                                />
-                                    <FormFeedback id="passwordConfirmationValidationMessage">
-                                    {!arePasswordsIdentical ? "New passwords must match." : !passwordMeetsRequirements && PASSWORD_REQUIREMENTS}
-                                    </FormFeedback>
-                            </FormGroup>
+                        <Col md={{ size: 6, offset: 3 }}>
+                        <FormGroup>
+                            <Label htmlFor="password-confirm">
+                            Re-enter new password
+                            </Label>
+                            <Password
+                                passwordFieldType="Confirm"
+                                isPasswordVisible={isPasswordVisible}
+                                setIsPasswordVisible={setIsPasswordVisible}
+                                invalid={!!currentPassword && !isNewPasswordConfirmed}
+                                onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+                                setNewPasswordConfirm(e.target.value);
+                                setMyUser(
+                                Object.assign({}, myUser, {
```

Comment:
Does this help? https://code.visualstudio.com/updates/v1_49#_only-format-modified-text

The current indentation of the modified lines makes it hard to read.

Reply:
That will be helpful going forward! Although it does modify the entire section of code, not just the modified lines - so now when I commit it will change everything in the return for the component - I'll do in its own commit.